advanced/readme_kiali.adoc

```diff
@@ -0,0 +1,183 @@
+istio-ingressgateway-54ddb65d74-sznf9       0/1       Running     0          1d
+istio-pilot-58c7c644c7-l88d7                0/2       Running     0          1d
+istio-policy-586559fdd9-l88v9               0/2       Running     0          1d
+istio-sidecar-injector-6fc85fbd55-6r2hj     0/1       Running     0          1d
+istio-statsd-prom-bridge-6dbb7dcc7f-7qmbw   0/1       Running     0          1d
+istio-telemetry-7988677557-tz4sm            0/2       Running     0          1d
+jaeger-agent-lmhbb                          0/1       Running     0          0s
+jaeger-collector-68fd846775-wll4r           0/1       Running     0          1d
+jaeger-query-58f4655965-q8jjk               0/1       Running     0          1d
+kiali-7966d98797-77gqz                      0/1       Running     0          1d
+prometheus-586d95b8d9-r6d6x                 0/1       Running     0          1d
+----
+
+If the "STATUS" column for Kiali is "Running", you can check which URL it is
+using by using this command:
+
+[source, bash]
+----
+NAME                   HOST/PORT                                                PATH      SERVICES               PORT              TERMINATION   WILDCARD
+grafana                grafana-istio-system.192.168.15.12.nip.io                          grafana                http                            None
+istio-ingress          istio-ingress-istio-system.192.168.15.12.nip.io                    istio-ingress          http                            None
+istio-ingressgateway   istio-ingressgateway-istio-system.192.168.15.12.nip.io             istio-ingressgateway   http                            None
+jaeger-query           jaeger-query-istio-system.192.168.15.12.nip.io                     jaeger-query           jaeger-query      edge          None
+kiali                  kiali-istio-system.192.168.15.12.nip.io                            kiali                  <all>                           None
+prometheus             prometheus-istio-system.192.168.15.12.nip.io                       prometheus             http-prometheus                 None
+tracing                tracing-istio-system.192.168.15.12.nip.io                          tracing                tracing           edge          None
+----
+
+So now we have the URL for Kiali (in my case,
+"kiali-istio-system.192.168.15.12.nip.io"), so let's access it:
+
+image::https://i.imgur.com/rG5tdZu.png[Login Page]
+
+This is the login page. The default credentials is "admin/admin", but it's
+recommended to change it before using it in production.
+
+== Service Graph
+
+After login, you should be greeted with the graph page:
+
+image::https://i.imgur.com/PtKVfe4.png[Graph Page]
+
+It shows a graph with all the microservices, connected by the requests going
+through then. On this page, you can see how the services interact with each
+other.
+
+== Service Listing
+
+In this page, you can get a listing of all the services that are running on the
```

Comment:
Click the Services link in the left navigation.  On the Services page you can view a listing of all the services that are running on the cluster, and additional information about them, such as health status.